gradlew.bat

```diff
@@ -1,89 +1,89 @@
-@rem
```

Comment:
@tiagobcx  any idea why this change ?

Reply:
Not sure, but I double checked and has the same content as main